cost/turbonomics/allocate_virtual_machines_recommendations/aws/turbonomics_allocate_virtual_machines.pt

```diff
@@ -0,0 +1,370 @@
+name "Turbonomics Allocate Virtual Machine Recommendations AWS"
+rs_pt_ver 20180301
+type "policy"
+short_description "Turbonomics policy for allocating RI coverage for virtual machines [README](https://github.com/flexera-public/policy_templates/tree/master/cost/turbonomics/allocate_virtual_machines_recommendations/aws) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+severity "low"
+category "Cost"
+default_frequency "daily"
+info(
+  version: "2.1",
+  provider: "AWS",
+  source: "Turbonomics",
+  service: "Compute",
+  policy_set: "Allocate VMs",
+  recommendation_type: "Rate Reduction",
+  publish: "false"
+)
+
+##################
+# Parameters   #
+##################
+
+parameter "auth_cookie" do
+  type "string"
+  label "Authorization Cookie"
+  no_echo true
+  description "authorization cookie pulled from manual source"
```

Comment:
Perhaps change description to "Valid authorization cookie"

Reply:
@taniaSol - Done!